cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/lambda/InvokeLambdaAction.java

```diff
@@ -0,0 +1,123 @@
+     * @param proxyHost         Optional - proxy server used to connect to Amazon API. If empty no proxy will be used.
+     *                          Default: ""
+     * @param proxyPort         Optional - proxy server port. You must either specify values for both proxyHost and
+     *                          proxyPort inputs or leave them both empty.
+     *                          Default: ""
+     * @param proxyUsername     Optional - proxy server user name.
+     *                          Default: ""
+     * @param proxyPassword     Optional - proxy server password associated with the proxyUsername input value.
+     *                          Default: ""
+     * @param function          Lambda function name to call
+     *                          Example: "helloWord"
+     * @param qualifier         Optional - Lambda function version or alias
+     *                          Example: ":1"
+     *                          Default: "$LATEST"
+     * @param payload           Optional - Lambda function payload in JSON format
+     *                          Example: "{"key1":"value1", "key1":"value2"}"
+     *                          Default: null
+     * @return                  A map with strings as keys and strings as values that contains: outcome of the action, returnCode of the
+     *                          operation, or failure message and the exception if there is one
+     */
+    @Action(name = "Invoke AWS Lambda Function",
+            outputs = {
+                    @Output(Outputs.RETURN_CODE),
+                    @Output(Outputs.RETURN_RESULT),
+                    @Output(Outputs.EXCEPTION)
+            },
+            responses = {
+                    @Response(text = Outputs.SUCCESS, field = Outputs.RETURN_CODE, value = Outputs.SUCCESS_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = Outputs.FAILURE, field = Outputs.RETURN_CODE, value = Outputs.FAILURE_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR)
+            }
+    )
+    public Map<String, String> execute(
+            @Param(value = IDENTITY,   required = true)  String identity,
+            @Param(value = CREDENTIAL, required = true, encrypted = true)  String credential,
+            @Param(value = REGION,     required = true)  String region,
+            @Param(value = PROXY_HOST)                   String proxyHost,
+            @Param(value = PROXY_PORT)                   int    proxyPort,
+            @Param(value = PROXY_USERNAME)               String proxyUsername,
+            @Param(value = PROXY_PASSWORD)               String proxyPassword,
+            @Param(value = FUNCTION_NAME, required = true) String function,
+            @Param(value = FUNCTION_QUALIFIER)          String qualifier,
```

Comment:
@lrevnic  Please add defaultIfEmpty(qualifier, DEFAULT_QUALIFIER) and set the value to $LATEST in order to avoid null inputs, in case none are provided from the .sl file.
Do the same for others that might have the same issue